doc/migrating/0.4-Migration.md

```diff
@@ -163,6 +174,17 @@ on(largeButtonNode, 'click', function () {
 });
 ```
 
+When you want to sort the displayed items by multiple properties, set `sort` to an array of objects where each object contains a property named `property` identifying the data field to sort, and optionally a descending property which, if true, indicates to sort descending rather than ascending.
+
+```js
+var grid = new OnDemandGrid({
+	store: store,
```

Comment:
This would be `collection` in 0.4+ ;)

(I'm fixing this now locally before merging)